modules/dxp/apps/search-experiences/search-experiences-web/src/main/resources/META-INF/resources/sxp_blueprint_admin/js/utils/fdsPropsTransformerActions.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {fetch} from 'frontend-js-web';
+
+import {DEFAULT_ERROR} from './errorMessages';
+import {DEFAULT_HEADERS} from './fetch/fetch_data';
+import {openErrorToast, openSuccessToast} from './toasts';
+
+export function download(url, parameters, title) {
+	fetch(url, parameters)
+		.then((response) => {
+			if (!response.ok) {
+				throw DEFAULT_ERROR;
+			}
+
+			return response.blob();
+		})
+		.then((responseBlob) => {
+			const downloadElement = document.createElement('a');
+
+			downloadElement.download = title + '.json';
+			downloadElement.href = URL.createObjectURL(responseBlob);
+
+			document.body.appendChild(downloadElement);
+
+			downloadElement.click();
+
+			openSuccessToast();
+		})
+		.catch(() => {
+			openErrorToast();
+		});
+}
+
+export function updateCollectionProvider({
+	collectionProvider = true,
+	itemData,
+	loadData,
+}) {
+	fetch(`/o/search-experiences-rest/v1.0/sxp-blueprints/${itemData.id}`, {
+		headers: {
+			...DEFAULT_HEADERS,
+			'X-Liferay-Accept-All-Languages': true,
+		},
+		method: 'GET',
+	})
```

Comment:
Can you add documentation above this about the reasoning behind doing another fetch here? It might look odd in the future so it'd be good to point out the `title_i18n` and `description_i18n` data.